### PR TITLE
chore: Publish crate to Crates.io

### PR DESCRIPTION
**Pull request by matchai**
#### Description

- Add all the necessary `Cargo.toml` details for crates.io
- Update the image on README.md to be a full URL so crates.io can load it
- Update installation instructions to encourage installing from crates.io

[📝 View updated README.md](https://github.com/starship/starship/tree/publish-crate)

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?

- [x] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,8 +1,22 @@
 [package]
 name = "starship"
-version = "0.1.0"
-authors = ["Matan Kushner <hello@matchai.me>"]
+version = "0.1.0-beta.4"
 edition = "2018"
```
Comment (johnletey) [inline Cargo.toml]: 2018? Not 2019?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+        <a href="https://crates.io/crates/starship"><img src="https://badgen.net/crates/v/starship" alt="Crates.io version"></a>
         <a href="https://dev.azure.com/starship-control/starship/_build"><img src="https://badgen.net/azure-pipelines/starship-control/starship/Starship%20Test%20Suite" alt="Azure Pipelines Build Status"></a>
         <a href="#contributors"><img src="https://badgen.net/badge/all%20contributors/7/orange" alt="All Contributors"></a>
         <a href="https://discord.gg/8Jzqu3T"><img src="https://badgen.net/badge/chat/on%20discord/7289da" alt="Chat on Discord"></a>
     </p>
 	<br>
 </h1>
 
-> ⚠️ This is very much work in progress, so please don't try to use it just yet!
+> ⚠️ Starship is still a work in progress, but is now far enough along to be used as a prompt (I use it as my daily driver)
```
Comment (johnletey) [inline README.md]:
```suggestion
> ⚠️ Starship is still a work in progress but is now far enough along to be used as a prompt (I personally use it as my daily driver)
```

File: Cargo.toml (hunk 1 above)
Comment (matchai) [reply Cargo.toml]: Yep. Rust edition 2018 was released at the end of last year and is still the latest edition.

Comment (johnletey) [reply Cargo.toml]: I see @matchai

**Pull request merged**